pkg/node/types/node_test.go

```diff
@@ -197,6 +197,9 @@ func (s *NodeSuite) TestNode_ToCiliumNode(c *C) {
 		IPv6IngressIP:           net.ParseIP("c0de::2"),
 		NodeIdentity:            uint32(12345),
 		WireguardPubKey:         "6kiIGGPvMiadJ1brWTVfSGXheE3e3k5GjDTxfjMLYx8=",
+		Annotations: map[string]string{
+			"cilium.io/bgp-virtual-router.64512": "router-id=172.0.0.3",
```

Comment:
Nit: it'd be better to update the test too with `annotation.BGPVRouterAnnoPrefix`.

Reply:
Using the prefix with a hardcoded test suffix now.